chall02/pcuadrad.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+import sys
+
+def check_string(txt):
+    for char in txt:
+        if not char.isalpha() and not char.isspace():
+            return 0
+    return 1
+
+def parse_morse(txt):
+    morse = {
+        "A": ".-", "B": "-...", "C": "-.-.", "D": "-..", "E": ".", "F": "..-.",
+        "G": "--.", "H": "....", "I": "..", "J": ".---", "K": "-.-", "L": ".-..",
+        "M": "--", "N": "-.", "O": "---", "P": ".--.", "Q": "--.-", "R": ".-.",
+        "S": "...", "T": "-", "U": "..-", "V": "...-", "W": ".--", "X": "-..-",
+        "Y": "-.--", "Z": "--..", " ": " "
+    }
+    str_morse = ""
+    for char in txt:
+        str_morse += morse.get(char)
+    return str_morse
```

Comment:
You could have made this more pythonic with this oneliner:
```suggestion
   return "".join(morse[c] for c in txt)
```